### DISCUSSION TITLE
今天是鸭鸭的生日耶

### DISCUSSION THREAD

**Discussion by NiuBoss123** [share121/inter-knot#2711] [其他]
![1809b567ee823bc741cd0d3e355c5891_3955607419716857496](https://github.com/user-attachments/assets/1e4e9032-0ff1-4aa1-bb4f-5a38993e2665)

![IMG_20240818_162909](https://github.com/user-attachments/assets/015271dd-ffa1-485b-9793-5ef4ea21e3ea)

布洛妮娅生日快乐🎉🎉🎉🎉🎉🎉

**Comment by Robei1** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >

**Comment by KongBai000-0** [comment]
问问：鸭全身哪是宝？

**Reply by huaxia2010** [reply]
扒鸭屁股是块宝！

**Comment by huihui88800** [comment]
鸭鸭全身都是宝
